### PR TITLE
minimal docs update

### PR DESCRIPTION
**Pull request by stefania11**
just editing landing page on docs for now

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -30,6 +30,7 @@ rich = "^12.6.0"
 dataclasses-json = "^0.5.7"
 validators = "^0.20.0"
 oauth2-client = "^1.3.0"
+cinder = "^21.1.0"
```
Comment (mdwelsh) [inline pyproject.toml:33]: mkdocs-cinder is already in the dev dependencies. It should not be needed by users of the SDK.

**Review by mdwelsh — APPROVED** [review]
LGTM but please remove cinder from the SDK dependencies, since I don't think it is needed.

**Comment by mdwelsh** [comment]
Note that poetry.lock needs to be checked into the source tree as it defines the actual dependencies used by a given checkout of the tree.

File: docs/index.md [hunk 2]
```diff
@@ -22,20 +22,20 @@ Install the Fixie SDK using pip:
 $ pip install fixieai
 ```
 
-Set your `FIXIE_API_KEY` environment variable to your API key, which
-you can find in the Fixie web interface:
-```shell
-$ export FIXIE_API_KEY=<YOUR API KEY>
-```
-
+Once installed, run `fixie auth` to authenticate with Fixie and start building.
```
Comment (zkoch) [inline docs/index.md:25]: I think we learned we need to tell them to make sure they've first authenticated and created an account on Fixie. Then they can auth

**Comment by stefania11** [comment]
Cinder is needed when you want to run `just serve-docs` and check the docs
locally

**Pull request closed without merging**